app/services/handover_date_service.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+class HandoverDateService
+  # rubocop:disable Metrics/LineLength
+  def self.handover_start_date(offender)
+    if offender.nps_case? && !offender.early_allocation?
+      return [offender.earliest_release_date - 8.months, 'NPS Indeterminate'] if SentenceTypeService.indeterminate_sentence?(offender.imprisonment_status)
```

Comment:
Better to have this as an if block rather than have the linelength exclusion for rubocop I think...